### ISSUE TITLE
Etherpad 1.8.9 installation fails under Windows 

### ISSUE THREAD

**Issue by TheEpicSnowWolf** [ether/etherpad-lite#4848]
**Describe the bug**
The installOnWindows.bat script fails to install Etherpad 1.8.9 under Windows Server 2019. Below is the output from the script. "Das System kann die angegebene Datei nicht finden" is German for basically "file not found". I originally tried updating from 1.8.7 this way, but then tried a fresh installation which also did not work.

```
C:\etherpad-lite>.\src\bin\installOnWindows.bat
_
Ensure that all dependencies are up to date...  If this is the first time you have run Etherpad please be patient.
symbolische Verknüpfung erstellt für ep_etherpad-lite <<===>> ..\src
Das System kann die angegebene Datei nicht finden.
npm WARN prepare removing existing node_modules/ before installation

> sqlite3@5.0.1 install C:\etherpad-lite\src\node_modules\sqlite3
> node-pre-gyp install --fallback-to-build

node-pre-gyp WARN Using request for node-pre-gyp https download
node-pre-gyp WARN Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/sqlite3/v5.0.1/napi-v6-win32-x64.tar.gz
node-pre-gyp WARN Pre-built binaries not found for sqlite3@5.0.1 and node@14.15.5 (node-v83 ABI, unknown) (falling back to source compile with node-gyp)
gyp ERR! find VS
gyp ERR! find VS msvs_version not set from command line or npm config
gyp ERR! find VS VCINSTALLDIR not set, not running in VS Command Prompt
gyp ERR! find VS could not use PowerShell to find Visual Studio 2017 or newer
gyp ERR! find VS looking for Visual Studio 2015
gyp ERR! find VS - not found
gyp ERR! find VS not looking for VS2013 as it is only supported up to Node.js 8
gyp ERR! find VS
gyp ERR! find VS **************************************************************
gyp ERR! find VS You need to install the latest version of Visual Studio
gyp ERR! find VS including the "Desktop development with C++" workload.
gyp ERR! find VS For more information consult the documentation at:
gyp ERR! find VS https://github.com/nodejs/node-gyp#on-windows
gyp ERR! find VS **************************************************************
gyp ERR! find VS
gyp ERR! configure error
gyp ERR! stack Error: Could not find any Visual Studio installation to use
gyp ERR! stack     at VisualStudioFinder.fail (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-visualstudio.js:121:47)
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-visualstudio.js:74:16
gyp ERR! stack     at VisualStudioFinder.findVisualStudio2013 (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-visualstudio.js:351:14)
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-visualstudio.js:70:14
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-visualstudio.js:372:16
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\util.js:54:7
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\util.js:33:16
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:315:5)
gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
gyp ERR! stack     at maybeClose (internal/child_process.js:1048:16)
gyp ERR! System Windows_NT 10.0.18362
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "configure" "--fallback-to-build" "--module=C:\\etherpad-lite\\src\\node_modules\\sqlite3\\lib\\binding\\napi-v6-win32-x64\\node_sqlite3.node" "--module_name=node_sqlite3" "--module_path=C:\\etherpad-lite\\src\\node_modules\\sqlite3\\lib\\binding\\napi-v6-win32-x64" "--napi_version=7" "--node_abi_napi=napi" "--napi_build_version=6" "--node_napi_label=napi-v6"
gyp ERR! cwd C:\etherpad-lite\src\node_modules\sqlite3
gyp ERR! node -v v14.15.5
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute 'C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js configure --fallback-to-build --module=C:\etherpad-lite\src\node_modules\sqlite3\lib\binding\napi-v6-win32-x64\node_sqlite3.node --module_name=node_sqlite3 --module_path=C:\etherpad-lite\src\node_modules\sqlite3\lib\binding\napi-v6-win32-x64 --napi_version=7 --node_abi_napi=napi --napi_build_version=6 --node_napi_label=napi-v6' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (C:\etherpad-lite\src\node_modules\node-pre-gyp\lib\util\compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:1048:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
node-pre-gyp ERR! System Windows_NT 10.0.18362
node-pre-gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\etherpad-lite\\src\\node_modules\\node-pre-gyp\\bin\\node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd C:\etherpad-lite\src\node_modules\sqlite3
node-pre-gyp ERR! node -v v14.15.5
node-pre-gyp ERR! node-pre-gyp -v v0.11.0
node-pre-gyp ERR! not ok
Failed to execute 'C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js configure --fallback-to-build --module=C:\etherpad-lite\src\node_modules\sqlite3\lib\binding\napi-v6-win32-x64\node_sqlite3.node --module_name=node_sqlite3 --module_path=C:\etherpad-lite\src\node_modules\sqlite3\lib\binding\napi-v6-win32-x64 --napi_version=7 --node_abi_napi=napi --napi_build_version=6 --node_napi_label=napi-v6' (1)

> wd@1.12.1 install C:\etherpad-lite\src\node_modules\wd
> node scripts/build-browser-scripts

added 683 packages in 15.157s
_
Clearing cache...
Das System kann die angegebene Datei nicht finden.
_
Setting up settings.json...
Can't find settings.json.
Copying settings.json.template...
Das System kann die angegebene Datei nicht finden.

C:\etherpad-lite\src>
```

**To Reproduce**
1. On Windows, perform a manual installation of Etherpad 1.8.9:
2. Run `git clone --branch master https://github.com/ether/etherpad-lite.git`
3. Run `src\bin\installOnWindows.bat`

**Server (please complete the following information):**
 - Etherpad version: 1.8.9
 - OS: Windows Server 2019
 - Node.js version: v14.15.5
 - npm version: v6.14.11
 - git version: v2.30.1.windows.1

**Comment by JohnMcLear** [comment]
Hey @TheEpicSnowWolf Did you try grabbing the windows .zip and using it that way?  Afaik git clone > installOnWindows.bat is an unusual workflow, the reason being is that if you are git cloning we expect you will use some sort of pm2 type thing to be able to run Etherpad.

TLDR; download the windows release zip and see if that works, don't use git clone.

Further reasoning: Windows needs symlinks enabling to function properly with Etherpad.

**Comment by TheEpicSnowWolf** [comment]
Hi @JohnMcLear, thanks for the quick reply! The pre-prepared windows package works fine. I was following the [installation instructions](https://github.com/ether/etherpad-lite/blob/master/README.md#windows) from the wiki and figured that a manual installation over git would make things easier in the long run since I could just pull the changes for a new version from git instead of handling the files manually (it's possible that this is a wrong assumption on my end though :D )

Either way, I still believe that there might be an issue with the installation script, since I previously installed Etherpad 1.8.7 successfully, also via the manual installation, and no parameters have changed on my system since then.

**Comment by JohnMcLear** [comment]
If you enable symlinks using git clone and pull will work but they are fragile and often break, it appears the windows implementation of symlinks is unpredictable.

**Comment by TheEpicSnowWolf** [comment]
Fair enough, then I'll probably just steer clear of the manual installation via git for now. Thanks for the advice! Maybe the installation guide should be updated to mention the potential issues?

**Comment by JohnMcLear** [comment]
Yes please, would you mind doing that please?  I appreciate it!

**Comment by TheEpicSnowWolf** [comment]
@JohnMcLear I created a pull request for this. #4861 Hope that's okay!

**Comment by JohnMcLear** [comment]
@TheEpicSnowWolf CI is now building a windows installer too which will be included in the next release

https://github.com/ether/etherpad-lite/suites/2116704273/artifacts/43121208 if you want to try it, before that expires 📄

**Comment by JohnMcLear** [comment]
nevermind, that file is broken 📦

**Comment by TheEpicSnowWolf** [comment]
I mean I was still able to download the file and tried it anyway. :D Installation worked fine, but it doesn't create an uninstaller what I think it should definitely do. Also I think it would be more user-friendly to give the user the option to start Etherpad after installation instead of doing it anyway,

**Comment by JohnMcLear** [comment]
I'd love to see a PR for the uninstaller :: https://github.com/ether/etherpad_nsis
